### PR TITLE
Add description to checkbox "Enable property inheritance" 

### LINKED ISSUE #24: Add description to checkbox "Enable property inheritance"
To most users it is not immediately obvious what the checkbox highlighted below means:
Screenshot 2024-03-06 at 19 59 52
It would be beneficial to add a descriptive text there adding some more context similar to <https://sling.apache.org/documentation/bundles/context-aware-configuration/context-aware-configuration-default-implementation.html#property-inheritance>.

### PR DESCRIPTION
**Pull request by stefanseifert**
Fixes #24

### CODE REVIEW

**Comment by stefanseifert** [comment]
@kwin here is a proposal for #24 which adds an "i" button with more text behind the "Enable property inheritance" checkbox.

not sure if the text i've put into the popup is helpful enough, maybe you have a better idea for the wording in english and german?

File: bundle/src/main/resources/i18n/de.json [hunk 1]
```diff
@@ -45,7 +45,8 @@
         "collection": "Listen-Vererbung aktivieren",
         "collectionItem": "Diese Konfiguration wird über Listen-Vererbung vererbt.",
         "config": "Diese Konfiguration wird vererbt.",
-        "property": "Eigenschafts-Vererbung aktivieren"
+        "property": "Eigenschafts-Vererbung aktivieren",
+        "propertyDescription": "Wenn aktiviert werden Eigenschaften die hier nicht explizit gesetzt sind von Konfigurations aus höheren Vererbungs-Ebenen übernommen."
```
Comment (kwin) [inline bundle/src/main/resources/i18n/de.json:49]:
```suggestion
        "propertyDescription": "Wenn aktiviert, werden Eigenschaften, die hier nicht explizit gesetzt sind, von Konfigurationen aus höheren Ebenen des Vererbungsbaums übernommen."
```

**Review by kwin — COMMENTED** [review]
Just one minor remark about the german translation.

File: bundle/src/main/resources/i18n/de.json (hunk 1 above)
Comment (stefanseifert) [reply bundle/src/main/resources/i18n/de.json]: thanks, i've used the better translation

**Pull request merged**